acquisitions/src/main/resources/thunderjet/mod-orders/features/routing-list-print-template.feature

```diff
@@ -199,4 +199,4 @@
       Given path 'orders/routing-lists/' + routingListId + '/template'
       When method GET
       Then status 200
-      And match response.map.result.body == '<div><p>Biography room</p> <p>Falcon Denesik - 113 Law</p><p>Artimus Denesik - 143 Law</p><p>John Huels - 188 Law</p><p>Some note</p></div>'
+      And match response.map.result.body == '<div><p>Biography room</p> <p>test</p> <p>Falcon Denesik - 113 Law</p><p>Artimus Denesik - 143 Law</p><p>John Huels - 188 Law</p><p>Some note</p></div>'
```

Comment:
it is better to set custom title in here orderLine.titleOrPackage = 'title 1'. It will be more clear where title is coming from
https://github.com/folio-org/folio-integration-tests/pull/1448/files#diff-4724ffeb6ddfc97929ce8dec7e750187e3844ffad5137a4491789c69aecb3c22R141

Reply:
improved